osu.Game/Online/API/ArchiveDownloadRequest.cs

```diff
@@ -10,15 +10,21 @@ public abstract class ArchiveDownloadRequest<TModel> : APIDownloadRequest
     {
         public readonly TModel Model;
 
-        public float Progress;
+        public double Progress { get; private set; }
```

Comment:
disagree. should have remained a `float`. there's no point having double precision for data as throwaway as this. now that we're on .NET 5 you could even make this a `half` and i wouldn't blink an eye.

Reply:
oh, good reason. will revert then.

Reply:
...this is still `double`?

Reply:
ah missed it (and it's coincidentally the one this conversation started at)